scripts/zones/Batallia_Downs/npcs/no_name_0.lua

```diff
@@ -0,0 +1,35 @@
+-----------------------------------
+-- Area: Batallia Downs
+--  NPC:
```

Comment:
Others, to clarify:
```
Area: Zone
 NPC: Name
```
or
```
Area: Zone
NPC : Name
```
?

Reply:
I named them all to Lycopodium Teleport since it gives understanding of what they are (there was two in one same map that added _0 and _1 at their end)

Reply:
> Others, to clarify:
> 
> ```
> Area: Zone
>  NPC: Name
> ```
> 
> or
> 
> ```
> Area: Zone
> NPC : Name
> ```
> 
> ?

no cares as long as we go and make all the scripts do the same thing (eventually). if they don't and I run into them the COD takes over..

Reply:
@TeoTwawki  changed it back to blank then as it makes more sense 🐱